src/platform/src/core/utils/proxyClient.js

```
@@ -42,8 +75,24 @@
       };
 
       // Handle request body for POST, PUT, PATCH
-      if (['POST', 'PUT', 'PATCH'].includes(req.method) && req.body) {
-        config.data = req.body;
+      if (['POST', 'PUT', 'PATCH'].includes(req.method)) {
+        if (context && context.params) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)